agent.py

```diff
@@ -35,14 +34,13 @@
 )
 from daemon import AgentSupervisor, Daemon
 from emitter import http_emitter
-from jmxfetch import JMXFetch, JMX_LIST_COMMANDS
 from util import (
     EC2,
     get_hostname,
-    get_os,
     Watchdog,
 )
 from utils.flare import configcheck, Flare
+from utils.jmx_command import jmx_command
```

Comment:
I agree, that makes a lot of sense, I'll regroup them in one `utils.jmx` module ;)